analyzers/src/SonarAnalyzer.CSharp/Rules/MultipleVariableDeclaration.cs

```diff
@@ -18,27 +18,17 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using System.Collections.Generic;
-using System.Collections.Immutable;
-using System.Linq;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.CSharp;
-using Microsoft.CodeAnalysis.CSharp.Syntax;
-using Microsoft.CodeAnalysis.Diagnostics;
-using SonarAnalyzer.Helpers;
-
 namespace SonarAnalyzer.Rules.CSharp
 {
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
-    public sealed class MultipleVariableDeclaration : MultipleVariableDeclarationBase<SyntaxKind,
-        FieldDeclarationSyntax, LocalDeclarationStatementSyntax>
+    public sealed class MultipleVariableDeclaration : MultipleVariableDeclarationBase<SyntaxKind, FieldDeclarationSyntax, LocalDeclarationStatementSyntax>
     {
-        private static readonly DiagnosticDescriptor rule =
-            DescriptorFactory.Create(DiagnosticId, MessageFormat);
-        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } = ImmutableArray.Create(rule);
+        private static readonly DiagnosticDescriptor Rule = DescriptorFactory.Create(DiagnosticId, MessageFormat);
+
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } = ImmutableArray.Create(Rule);
 
-        public override SyntaxKind FieldDeclarationKind => SyntaxKind.FieldDeclaration;
-        public override SyntaxKind LocalDeclarationKind => SyntaxKind.LocalDeclarationStatement;
+        protected override SyntaxKind FieldDeclarationKind => SyntaxKind.FieldDeclaration;
```

Comment:
Once the generic base is inherited, FieldDeclaration line can be removed (is available via Language.SyntaxKind.FieldDeclaration).
And I would use the opportunity to remove `LocalDeclarationStatement` too and add it to facade